### PR TITLE
lightning implemented

### PR DESCRIPTION
**Pull request by neelm88**

### CODE REVIEW

**Comment by Abyszero** [comment]
Is there already a validation set or do I have to make one? If the latter, what do you think of doing k-folds instead? Then, the split % could be an input to `create_data_loader` and that function can just return two DataLoader's in that case.
